core/src/main/java/com/oath/oak/BlockMemoryAllocator.java

```diff
@@ -13,7 +13,7 @@
  * any thread.
  * Note that two allocations cannot be merged into a single allocation, and an allocation currently is not split.
  */
-interface OakBlockMemoryAllocator {
+interface BlockMemoryAllocator {
```

Comment:
This was initially an outside interface... Before the package was removed, so it is a candidate to be exposed outside in the future...